app/src/organisms/SendProtocolToOT3Slideout/index.tsx

```diff
@@ -152,7 +150,7 @@ export function SendProtocolToOT3Slideout(
           onClick={handleSendClick}
           width="100%"
         >
-          {t('shared:proceed_to_setup')}
+          {t('protocol_details:send')}
```

Comment:
should this be `protocol_list`?  Also, can you capitalize it through the i18n formatter function instead of directly in the i18n value? Since the string is 1 word long, its better to keep it all lower case to make it more reusable. You can look at the different formatter functions we have here: `app/src/i18n.ts`

```suggestion
          {i18n.format(t('protocol_list:send'), 'capitalize')}
```

Reply:
Oh it looks like there is already a `send` i18n key in `protocol_details.json` - feel free to use that instead but please delete it from `protocol_list`!